File: Sentinel LDK 7.6/Samples/Runtime/csharp/detach/Product.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace detach_cs
{
    class Product
    {
        private String id;
        private String name;

        public Product(String aId, String aName)
        {
            id = aId;
            name = aName;
        }

        public String getName()
        {
            return name;
        }

        public String getId()
        {
            return id;
        }

        public override String ToString()
        {
            return name + " (" + id + ")";
        }
    }
}

File: Sentinel LDK 7.6/Samples/Runtime/vb.net/detach/detach_vb/Product.vb
Imports System

Namespace detach_vb
    Friend Class Product
        ' Methods
        Public Sub New(ByVal aId As String, ByVal aName As String)
            Me.id = aId
            Me.name = aName
        End Sub

        Public Function getId() As String
            Return Me.id
        End Function

        Public Function getName() As String
            Return Me.name
        End Function

        Public Overrides Function ToString() As String
            Return (Me.name & " (" & Me.id & ")")
        End Function


        ' Fields
        Private id As String
        Private name As String
    End Class
End Namespace

